packages/default/scss/badge/_variables.scss

```diff
@@ -1,50 +1,80 @@
+/// The width of the border around the Badge.
 /// @group badge
 $kendo-badge-border-width: 1px !default;
 
-/// Border radius of the badge.
+/// The border radius of the Badge.
 /// @group badge
 $kendo-badge-border-radius: $kendo-border-radius-md !default;
 
 
-/// Horizontal padding of the badge.
+/// The horizontal padding of the Badge.
 /// @group badge
 $kendo-badge-padding-x: k-map-get( $kendo-spacing, 1 ) !default;
+/// The horizontal padding of the small Badge.
+/// @group badge
 $kendo-badge-sm-padding-x: k-map-get( $kendo-spacing, 0.5 ) !default; // sass-lint:disable-line leading-zero
+/// The horizontal padding of the medium Badge.
+/// @group badge
 $kendo-badge-md-padding-x: k-map-get( $kendo-spacing, 1 ) !default;
+/// The horizontal padding of the large Badge.
+/// @group badge
 $kendo-badge-lg-padding-x: k-map-get( $kendo-spacing, 1.5 ) !default;
 
-/// Vertical padding of the badge.
+/// The vertical padding of the Badge.
 /// @group badge
 $kendo-badge-padding-y: k-map-get( $kendo-spacing, 1 ) !default;
+/// The vertical padding of the small Badge.
+/// @group badge
 $kendo-badge-sm-padding-y: k-map-get( $kendo-spacing, 0.5 ) !default; // sass-lint:disable-line leading-zero
+/// The vertical padding of the medium Badge.
+/// @group badge
 $kendo-badge-md-padding-y: k-map-get( $kendo-spacing, 1 ) !default;
+/// The vertical padding of the large Badge.
+/// @group badge
 $kendo-badge-lg-padding-y: k-map-get( $kendo-spacing, 1.5 ) !default;
 
-/// Font sizes of the badge.
+/// The font sizes of the Badge.
 /// @group badge
 $kendo-badge-font-size: $kendo-font-size-xs !default;
+/// The font size of the small Badge.
+/// @group badge
 $kendo-badge-sm-font-size: $kendo-font-size-xs !default;
+/// The font size of the medium Badge.
+/// @group badge
 $kendo-badge-md-font-size: $kendo-font-size-xs !default;
+/// The font size of the large Badge.
+/// @group badge
 $kendo-badge-lg-font-size: $kendo-font-size-xs !default;
 
-/// Line heights used along with $kendo-font-size.
+/// The line heights used along with $kendo-font-size.
 /// @group badge
 $kendo-badge-line-height: 1 !default;
+/// The line height used along with $kendo-font-size of the small Badge.
+/// @group badge
 $kendo-badge-sm-line-height: $kendo-badge-line-height !default;
+/// The line height used along with $kendo-font-size of the medium Badge.
+/// @group badge
 $kendo-badge-md-line-height: $kendo-badge-line-height !default;
+/// The line height used along with $kendo-font-size of the large Badge.
+/// @group badge
 $kendo-badge-lg-line-height: $kendo-badge-line-height !default;
 
-/// Badge min width used for circle badge.
+/// The calculated minimum width of the Badge used for circular Badge.
 /// @group badge
 $kendo-badge-min-width: calc( #{$kendo-badge-line-height * 1em} + #{$kendo-badge-padding-y * 2} + #{$kendo-badge-border-width * 2}) !default;
+/// The calculated minimum width of the small Badge used for circular Badge.
```

Comment:
As these descriptions repeat those in the `docs` folder, apply the same suggestions.